### ISSUE TITLE
help

### ISSUE THREAD

**Issue by 1212Miracle** [ykskks/Matrix-Factorization-for-Drug-Side-Effect-Prediction#3]
In the code for "make_severe_side_effects_list", I did not find "processed_sider. csv" in the dataset file.

**Comment by ykskks** [comment]
Hi, thanks for the comment.
I'm sorry I didn't notice till now.
I just uploaded the file.

**Comment by 1212Miracle** [comment]
Hello writer, sorry to disturb you again
In the code file "make_teain_dataset", the last mentioned "faers_19q2_data_dup. pkl", "all_keggid. pkl", "kegg_faers_proai. pkl", "same_drug_set_by_id. pkl", and "sever_adr_dics. pkl" cannot be found in the folder.

**Comment by ykskks** [comment]
Hi! 
In order to run make_train_dataset.py, you first need to run other .py files to generate .pkl files. (Please see data.sh.)
```
python ./src/data/clean_drug_reac.py
python ./src/data/make_drugname_to_keggid_mapping.py
python ./src/data/make_keggid_to_proai_mapping.py
python ./src/data/make_equivalent_drug_set.py
python ./src/data/make_severe_side_effects_list.py
```

These will generate necessary .pkl files to run
```
python ./src/data/make_train_dataset.py
python ./src/data/make_test_dataset.py
```

Also, the easiest way is to run `bash data.sh`. This will run all preprocessing scripts in the right order.

**Comment by 1212Miracle** [comment]
Thank you for your patience and tolerance towards a novice.
I have one final question.Is the program running in a Linux environment? Will there be many obstacles to running this program in a window environment?

**Comment by ykskks** [comment]
No problem!

The program was run and tested on Linux (Ubuntu). If you have access to Linux environment, I would recommend switching.

I am not sure about the Windows environment but as long as the requirements in the requirements.txt file is met, there should be not so many problems...

**Comment by 1212Miracle** [comment]
Hello writer，sorry to disturb you again and again.

In python ./src/data/clean_drug_reac.py , are the "./data/drug/*" and "./data/reac/*" mentioned in lines 66 and 67 non uploaded folders?

**Comment by ykskks** [comment]
Yes, as described in README and #2, you need to download tables from FAERS.

**Comment by 1212Miracle** [comment]
Hello author,
 In the code file 'make_ drugname_ to_ keggid_ in mapping. py ', it is necessary to go to' http://rest.kegg.jp/list/ndc' and 'http://rest.kegg.jp/link/ndc/drug' to reading data, but they are currently inaccessible. Do you have any new websites available?
Thank you for your help.
